lib/pinyin.js

```diff
@@ -144,6 +145,13 @@ class Pinyin {
         return PHONETIC_SYMBOL[$1_phonetic].replace(RE_TONE2, "$1");
       });
 
+    case PINYIN_STYLE.CAMEL:
+      var normal=pinyin.replace(RE_PHONETIC_SYMBOL, function($0, $1_phonetic){
+        return PHONETIC_SYMBOL[$1_phonetic].replace(RE_TONE2, "$1");
+      });
+	  return normal.charAt(0).toUpperCase()+normal.slice(1);
```

Comment:
我遇到了一点问题，能告诉我一下你用的eslint是什么版本的吗？

/home/intijk/workspace/pinyin/lib/index.js
  1:1  error  Rule 'no-empty-label' was removed and replaced by:
no-labels                 no-empty-label
  1:1  error  Rule 'no-extra-strict' was removed and replaced by:
strict                   no-extra-strict
  1:1  error  Rule 'no-wrap-func' was removed and replaced by:
no-extra-parens             no-wrap-func
  1:1  error  Rule 'space-return-throw-case' was removed and replaced by:
keyword-spacing  space-return-throw-case


2016-04-18 23:55 GMT-05:00 闲耘™ <notifications@github.com>:

> In lib/pinyin.js
> <https://github.com/hotoo/pinyin/pull/89#discussion_r60173209>:
>
> > @@ -144,6 +145,13 @@ class Pinyin {
> >          return PHONETIC_SYMBOL[$1_phonetic].replace(RE_TONE2, "$1");
> >        });
> >
> > +    case PINYIN_STYLE.CAMEL:
> > +      var normal=pinyin.replace(RE_PHONETIC_SYMBOL, function($0, $1_phonetic){
> > +        return PHONETIC_SYMBOL[$1_phonetic].replace(RE_TONE2, "$1");
> > +      });
> > +	  return normal.charAt(0).toUpperCase()+normal.slice(1);
>
> 代码风格请修正下。
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hotoo/pinyin/pull/89/files/0f410115686a1f7a9beb0416ccedc3f84098813a#r60173209>
>